.github/workflows/integration-test.yml

```
@@ -4,6 +4,20 @@ on:
   push:
     branches:
       - main
+  pull_request:
+    paths:
+      - '.github/workflows/integration-test.yml'
+      - 'go.mod'
+      - 'go.sum'
+      - 'cmd/otelcol/**'
+      - 'internal/**'
+      - '!internal/buildscripts/packaging/choco/**'
+      - '!internal/buildscripts/packaging/fpm/**'
+      - '!internal/buildscripts/packaging/installer/**'
+      - '!internal/buildscripts/packaging/msi/**'
+      - '!internal/buildscripts/packaging/release/**'
+      - '!internal/buildscripts/packaging/tests/**'
```

Would a more general `'!internal/buildscripts/**'` be wanted? I'm not sure if `update-deps` is worth running these for instance.

---

Unfortunately, I think we need to include `internal/buildscripts/packaging/collect-libs.sh` and `internal/buildscripts/packaging/smart-agent-release.txt` for the docker image build.  We can move these files to another dir and update the build scripts in a future PR?

---

More a question than a suggestion, but if we find this runs unnecessarily we can update as you suggest.

---

Maybe we can just use:
```suggestion
      - '!internal/buildscripts/*'
      - '!internal/buildscripts/packaging/*/**'
```
This should ignore everything currently in `internal/buildscripts/` except those 2 files.